File: app/src/main/java/com/example/bmicalculator/SplashActivity.java
package com.example.bmicalculator;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;

public class SplashActivity extends AppCompatActivity {
    private Button bmiButton, caloriesButton, recipesButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        bmiButton = findViewById(R.id.bmiButton);
        caloriesButton = findViewById(R.id.caloriesButton);
        recipesButton = findViewById(R.id.recipesButton);

        bmiButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(SplashActivity.this, BmiActivity.class);
                startActivity(intent);
            }
        });

        caloriesButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(SplashActivity.this, CaloriesActivity.class);
                startActivity(intent);
            }
        });

        recipesButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(SplashActivity.this, RecipesActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: Lab4/app/src/main/java/com/example/lab4/SplashActivity.kt
package com.example.lab4

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class SplashActivity : AppCompatActivity() {
    private lateinit var bmiButton: Button
    private lateinit var caloriesButton: Button
    private lateinit var recipesButton: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        bmiButton = findViewById(R.id.bmiButton)
        caloriesButton = findViewById(R.id.caloriesButton)
        recipesButton = findViewById(R.id.recipesButton)

        bmiButton.setOnClickListener {
            startActivity(Intent(this, BmiActivity::class.java))
        }

        caloriesButton.setOnClickListener {
            startActivity(Intent(this, CaloriesActivity::class.java))
        }

        recipesButton.setOnClickListener {
            startActivity(Intent(this, RecipesActivity::class.java))
        }
    }
}
